test/Table_Tests/src/Common_Table_Operations/Column_Operations_Spec.enso

```diff
@@ -749,11 +1108,18 @@ spec setup =
                 t.at "a" . zip (t.at "b") [_, _] . name . should_equal "[a] x [b]"
 
     Test.group prefix+"Column.rename" <|
-        Test.specify "should not allow illegal names" <|
-            t = table_builder [["a", [1, 2, 3]]]
-            c = t.at "a"
-
-            c.rename Nothing . should_fail_with Illegal_Argument
-            c.rename '' . should_fail_with Illegal_Argument
-            c.rename 'a\0b' . should_fail_with Illegal_Argument
-            c.rename '\0' . should_fail_with Illegal_Argument
+        Test.specify "Should allow the creation of constant columns" <|
+            t = table_builder [["x", ["1", "2", "3"]]]
```

Comment:
Something is wrong here.

`Should allow the creation of constant columns` seems to be in the group `Column.rename` but it does not belong there. And why were the naming tests removed??

Reply:
Oops, that was an accident.